consensus/consensus.go

```diff
@@ -140,6 +143,29 @@ func (consensus *Consensus) UpdateConsensusID(consensusID uint32) {
 	}
 }
 
+// WaitForNewRandomness listens to the RndChannel to receive new VDF randomness.
+func (consensus *Consensus) WaitForNewRandomness() {
```

Comment:
can we add randomness as a service? should be decoupled from calling from consensus service?

Reply:
Randomness is already a separate service. But consensus needs to wait for output from randomness service